lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -148,21 +150,25 @@
                         endDate = new Date(end);
 
                     if (pacingType === 'self_paced') {
-                        dateString = 'Self-paced';
-                        if (start && startDate > now) {
-                            dateString += ' - Starts ' + start;
+                        dateString = gettext('(Self-paced) ');
+                        if (start) {
+                            dateString += startDate > now ?
+                                StringUtils.interpolate(gettext('Starts {start}'), { start: start }) :
+                                StringUtils.interpolate(gettext('Started {start}'), { start: start });
                         } else if (end && endDate > now) {
-                            dateString += ' - Ends ' + end;
+                            StringUtils.interpolate(gettext('Ends {end}'), { end: end });
                         } else if (end && endDate < now) {
-                            dateString += ' - Ended ' + end;
+                            StringUtils.interpolate(gettext('Ended {end}'), { end: end });
                         }
                     } else {
                         if (start && end) {
                             dateString = start + ' - ' + end;
                         } else if (start) {
-                            dateString = 'Starts ' + start;
+                            dateString += startDate > now ?
+                                StringUtils.interpolate(gettext('Starts {start}'), { start: start }) :
+                                StringUtils.interpolate(gettext('Started {start}'), { start: start });
                         } else if (end) {
-                            dateString = 'Ends ' + end;
+                            StringUtils.interpolate(gettext('Ends {end}'), { end: end });
```

Comment:
Shouldn't this be assigned to a variable or something?

Reply:
Good catch, fixed.